plot_app/configured_plots.py

```diff
@@ -573,12 +573,12 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page):
                          'Battery Current [A]', 'Discharged Amount [mAh / 100]',
                          'Battery remaining [0=empty, 10=full]'])
     data_plot.change_dataset('system_power')
-    if data_plot.dataset and 'voltage5V_v' in data_plot.dataset.data:
-        if np.amax(data_plot.dataset.data['voltage5V_v']) > 0.0001:
-            data_plot.add_graph(['voltage5V_v'], colors8[7:8], ['5 V'])
+    if data_plot.dataset and 'voltage5v_v' in data_plot.dataset.data:
```

Comment:
This breaks backwards compatibility. Instead of breaking it, please add a new section for the new format so that both work. Please also add comments with a date, so that we can remove the dual-support later and know which one was the old format.

Reply:
Makes sense. Will do

Reply:
Yes, please add the checks to https://github.com/PX4/flight_review/blob/master/plot_app/configured_plots.py#L36, to check if it's using the old name. If so, rename the field into the new format, so that the new format is used throughout the code.

Reply:
@bkueng how do you suggest to add that check in L36 without the `data_plot` being init first?

Reply:
You will need to modify `ulog.data_list`. I can look into it if you want me to.